### PR TITLE
Update create.py

### LINKED ISSUE #5280: tag_create Documentation needs update
### CKAN Version if known (or site URL)
2.8.3

### The Issue
The tag_create API documentation has 2 issues:
1. For the tag name space is an allowed character but is not currently listed

2. The example of using the vocabulary name ('Genre') is incorrect the actual id is what is required. Issue #2362 sought a fix the software to match the documentation but was not completed. Unless it is doing to get fixed the documentation should be updated.

### PR DESCRIPTION
**Pull request by Gauravp-NEC**
Fix for issue 5280

Fixes #5280 

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by Gauravp-NEC** [comment]
@amercader , In this PR I have update the document that in tag name "space" is an allowed character.  Secondly the statement "the id of the vocabulary that the new tag should be added to, e.g. the id of vocabulary 'Genre'  " is correct as it clearly states that in the end " e.g. the id of vocabulary 'Genre' ", Where 'Genre' is just an example. It can be id of any other vocabulary as well which has been created.

**Comment by Gauravp-NEC** [comment]
@amercader , Please merge the PR if it looks good. Thankyou

**Comment by Gauravp-NEC** [comment]
@amercader , Please review this PR. If it looks good please merge. Thankyou.

File: ckan/logic/action/create.py [hunk 1]
```diff
@@ -1302,8 +1302,8 @@ def tag_create(context, data_dict):
     :py:func:`~ckan.logic.action.update.package_update` function.)
 
     :param name: the name for the new tag, a string between 2 and 100
-        characters long containing only alphanumeric characters and ``-``,
-        ``_`` and ``.``, e.g. ``'Jazz'``
+        characters long containing only alphanumeric characters, three punctuations i.e. ``-``,
+        ``_`` and ``.``and space. e.g. ``'Jazz'``
```
Comment (amercader) [inline ckan/logic/action/create.py:1306]: @Gauravp-NEC I think this reads better:

```suggestion
        characters long containing only alphanumeric characters, spaces and the characters ``-``,
        ``_`` and ``.``, e.g. ``'Jazz'``
```

**Comment by Gauravp-NEC** [comment]
@amercader , Thankyou for your suggestion. I have updated the PR in commit https://github.com/ckan/ckan/pull/5721/commits/e1be6b2c985228e1facbb0ec98c7bb3a3ee2963a

File: ckan/logic/action/create.py [hunk 2]
```diff
@@ -1302,7 +1302,7 @@ def tag_create(context, data_dict):
     :py:func:`~ckan.logic.action.update.package_update` function.)
 
     :param name: the name for the new tag, a string between 2 and 100
-        characters long containing only alphanumeric characters and ``-``,
+        characters long containing only alphanumeric characters, spaces and the characters ``-``,
```
Comment (amercader) [inline ckan/logic/action/create.py:1305]: Sorry @Gauravp-NEC, one more thing to make the tests pass:

```suggestion
        characters long containing only alphanumeric characters, 
        spaces and the characters ``-``,
```

**Comment by Gauravp-NEC** [comment]
@amercader , I have again updated this PR in commit https://github.com/ckan/ckan/pull/5721/commits/e12e8216f40024330b5066206ff4273815ac6647. Please merge this PR if it looks good now.

**Comment by Gauravp-NEC** [comment]
@amercader , I have update this PR in commit https://github.com/ckan/ckan/pull/5721/commits/e12e8216f40024330b5066206ff4273815ac6647. Please let me know if anything is remaining to do in this PR.

**Comment by Gauravp-NEC** [comment]
@amercader , Did the below changes looks better than commit https://github.com/ckan/ckan/pull/5721/commits/e12e8216f40024330b5066206ff4273815ac6647? If you agree I will commit the changes again. Please let me know your opinion. Thankyou

snap

**Comment by Gauravp-NEC** [comment]
@amercader , Did the below changes looks better than commit https://github.com/ckan/ckan/pull/5721/commits/e12e8216f40024330b5066206ff4273815ac6647 ? If you agree I will commit the changes again. Please let me know your opinion. Thankyou

snap

**Comment by amercader** [comment]
Thanks @Gauravp-NEC this is fine as is

**Pull request merged**